Runtime/Scripts/ExampleRest.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections;
+using UnityEngine;
+using UnityEngine.Networking; // for Unity Web Request
+
+using MobiledgeX;
+using DistributedMatchEngine;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System;
+
+    [RequireComponent(typeof(MobiledgeX.LocationService))]
+    public class ExampleRest : MonoBehaviour
+    {
+        MobiledgeXIntegration mxi;
+
+        IEnumerator Start()
+        {
+            yield return StartCoroutine(MobiledgeX.LocationService.EnsureLocation());
+            GetEdgeConnection();
+        }
+
+        async void GetEdgeConnection()
+        {
+            mxi = new MobiledgeXIntegration();
+            try
+            {
+                await mxi.RegisterAndFindCloudlet();
+            }
+            catch (DmeDnsException)
+            {
+                mxi.UseWifiOnly(true);
+                await mxi.RegisterAndFindCloudlet();
+            }
+            mxi.GetAppPort(LProto.L_PROTO_HTTP);
+            string url = mxi.GetUrl("http");
+            Debug.Log("Rest URL is : " + url);
+
```

Comment:
// Tiny comment about: do something with the new edge server URL "here"

Reply:
done